packages/workbox-cacheable-response/CacheableResponsePlugin.mjs

```diff
@@ -22,12 +23,21 @@ import './_version.mjs';
  */
 class CacheableResponsePlugin {
   /**
-   * @param {CacheableResponse} cacheableResponse A configured
-   * `CacheableResponse` instance.
+   * To construct a new CacheableResponsePlugin instance you must provide at
+   * least one of the `config` properties.
+   *
+   * If both `statuses` and `headers` are specified, then both conditions must
+   * be met for the `Response` to be considered cacheable.
+   *
+   * @param {Object} config
+   * @param {Array<number>} [config.statuses] One or more status codes that a
+   * `Response` can have and be considered cacheable.
+   * @param {Object<string,string>} [config.headers] A mapping of header names
+   * and expected values that a `Response` can have and be considered cacheable.
+   * If multiple headers are provided, only one needs to be present.
    */
-  constructor(cacheableResponse) {
-    this._cacheableResponse = cacheableResponse;
-    this.cacheWillUpdate = this.cacheWillUpdate.bind(this);
+  constructor(config = {}) {
```

Comment:
Plugin constructor appears to have been changed to not require the cacheable response as input per Matt's request. This change LGTM.